frontend/app/src/components/helper/AssetDetailsBase.vue

```diff
@@ -1,14 +1,22 @@
 <template>
   <list-item
-    v-bind="$attrs"
+    v-bind="rootAttrs"
     :class="opensDetails ? 'asset-details-base--link' : null"
     :dense="dense"
     :title="asset.isCustomAsset ? name : symbol"
     :subtitle="asset.isCustomAsset ? asset.customAssetType : name"
     @click="navigate"
   >
     <template #icon>
+      <v-img
+        v-if="asset.imageUrl"
+        contain
+        height="26px"
+        width="26px"
+        :src="asset.imageUrl"
```

Comment:
maybe need to put max width here? 
![Uploading image.png…]()

Reply:
Interesting, and it only happens when you remove the cursor